gradle/libs.versions.toml

```diff
@@ -242,3 +248,5 @@ ksp = { id = "com.google.devtools.ksp", version.ref = "ksp" }
 ktlint = { id = "org.jlleitschuh.gradle.ktlint", version.ref = "ktlintGradle" }
 mockposable = { id = "com.jeppeman.mockposable", version = "0.15" }
 paparazzi = { id = "app.cash.paparazzi", version = "2.0.0-alpha02" }
+android-application = { id = "com.android.application", version.ref = "agp" }
+kotlin-android = { id = "org.jetbrains.kotlin.android", version.ref = "kotlin" }
```

Comment:
these additional plugins aren't being used and can be removed.

I'm guessing they were auto-generated by the AS wizard